src/integration/java/org/humancellatlas/ingest/submission/web/ProjectStatusUpdateTest.java

```diff
@@ -0,0 +1,88 @@
+package org.humancellatlas.ingest.submission.web;
+
+import org.assertj.core.api.Assertions;
+import org.humancellatlas.ingest.config.MigrationConfiguration;
+import org.humancellatlas.ingest.project.Project;
+import org.humancellatlas.ingest.project.ProjectRepository;
+import org.humancellatlas.ingest.project.WranglingState;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.data.mongo.AutoConfigureDataMongo;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureDataMongo()
+@AutoConfigureMockMvc()
+public class ProjectStatusUpdateTest {
+    @Autowired
+    private MockMvc webApp;
+
+    @Autowired
+    private ProjectRepository projectRepository;
+
+    // NOTE: Adding MigrationConfiguration as a MockBean is needed as otherwise MigrationConfiguration won't be
+    //       initialised. This is very un-elegant and should be fixed.
+    @MockBean
+    private MigrationConfiguration migrationConfiguration;
```

Comment:
THANK YOU, this solves a problem I was having for one of my own integration tests.